### ISSUE TITLE
What's the learning rate update policy?

### ISSUE THREAD

**Issue by crazylyf** [marvis/pytorch-mobilenet#6]
Hi all, 

I tried to training MobileNet on my own, but the Prec@1 stops to increase at ~0.6. Am I missing something, what's the learning rate update policy should I use?
